packages/dokan.vm/dokan.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>dokan.vm</id>
-    <version>2.3.0</version>
+    <version>2.3.0.20250505</version>
     <authors>kacos2000</authors>
-    <description>Dokan simplifies the creation of custom file systems on Windows without the complexity of developing kernel-level drivers, offering an accessible solution for file system development, similar to FUSE on Linux.</description>
+    <description>Dokan is a user mode file system library for Windows with FUSE Wrapper.</description>
```

I think this description is not accurate. I believe FUSE is Linux file system in user space and does not exist in Windows.
```suggestion
    <description>Dokan simplifies Windows custom file system creation, similar to Linux's FUSE, without requiring device drivers.</description>
```